src/compas_fab/robots/robot.py

```diff
@@ -1601,26 +1599,21 @@ def transformed_axes(self, configuration, group=None):
     # drawing
     # ==========================================================================
 
-    def update(self, configuration, group=None, visual=True, collision=True):
+    def update(self, joint_state, visual=True, collision=True):
```

Comment:
Why don't we keep `configuration` as the parameter here, and do the conversion to `joint_states`  in this method, because that is needed by the artists; but then the usage for users stays unchanged.

Reply:
Not only it stays unchanged, but also, it's much easier to pass a configuration than to pass this zipped lists of joint states.